modules/sdp/SDP.js

```diff
@@ -353,6 +353,12 @@ SDP.prototype.toJingle = function(elem, thecreator) {
                 // TODO: handle params
                 elem.up();
             }
+
+            if (SDPUtil.findLine(this.media[i], 'a=extmap-allow-mixed', this.session)) {
+                elem.c('extmap-allow-mixed', {
+                    xmlns: 'urn:xmpp:jingle:apps:rtp:rtp-hdrext:0'
+                });
```

Comment:
Do you need an `up()` here?

Reply:
Yes.  (I don't think this is sufficient to fix the bugs tho?)